ingestion/src/metadata/cmd.py

```diff
@@ -123,4 +131,80 @@ def report(config: str) -> None:
         ) from exc
 
 
+@metadata.command()
+@click.option(
+    "-t",
+    "--type",
+    help="Workflow config",
+    required=True,
+)
+@click.option(
+    "-p",
+    "--path",
+    help="Workflow config",
+    required=False,
+)
+def docker_run(type: str, path: str = "") -> None:
+    """
+    Checks Docker Memory Allocation
+    Run Local Docker - metadata docker-run -t local
+    Run Latest Release Docker - metadata docker-run -t latest
+    """
+    try:
+        client = docker.from_env()
```

Comment:
this can throw an error if the docker is not running, we should catch that and let the user know
`except docker.errors.DockerException as error:`

its better to move this into a common method